pkg/authn/authsvc/authn.go

```diff
@@ -42,5 +42,5 @@ func (a authentication) Authenticate(ctx context.Context, token string) (authn.S
 	if err != nil {
 		return authn.Session{}, errors.Wrap(errors.ErrAuthentication, err)
 	}
-	return authn.Session{DomainUserID: res.GetId(), UserID: res.GetUserId(), DomainID: res.GetDomainId()}, nil
+	return authn.Session{ID: res.GetId(), UserID: res.GetUserId(), DomainID: res.GetDomainId()}, nil
 }
```

Comment:
Lets have seperate RPC call IdentifyPAT, move the logic to here

Something like below code 
```suggestion
	switch {
	case strings.HasPrefix(token, patPrefix+patSecretSeparator):
		res, err := a.authSvcClient.AuthenticatePAT(ctx, token)
		if err != nil {
			return authn.Session{}, errors.Wrap(errors.ErrAuthentication, err)
		}
		return authn.Session{ID: res.GetId(), UserID: res.GetUserId(), DomainID: res.GetDomainId()}, nil
	default:
		res, err := a.authSvcClient.Authenticate(ctx, &grpcAuthV1.AuthNReq{Token: token})
		if err != nil {
			return authn.Session{}, errors.Wrap(errors.ErrAuthentication, err)
		}
		return authn.Session{ID: res.GetId(), UserID: res.GetUserId(), DomainID: res.GetDomainId()}, nil
	}

```